packages/extension/src/App.tsx

```diff
@@ -94,136 +94,138 @@ const App: FC = () => {
-        <Route path={CREATE_NEW_WALLET_PATH} element={<CreateWallet />} />
-        <Route
-          path={HOME_PATH}
-          element={
-            <PrivateRoute>
-              <Home />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={LIST_WALLETS_PATH}
-          element={
-            <PrivateRoute>
-              <ListWallets />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={`${EDIT_WALLET_PATH}/:publicAddress`}
-          element={
-            <PrivateRoute>
-              <EditWallet />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={ADD_NEW_WALLET_PATH}
-          element={
-            <PrivateRoute>
-              <AddNewWallet />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={ADD_NEW_TRUSTLINE_PATH}
-          element={
-            <PrivateRoute>
-              <AddNewTrustline />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={TRANSACTION_PATH}
-          element={
-            <PrivateRoute>
-              <Transaction />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={HISTORY_PATH}
-          element={
-            <PrivateRoute>
-              <History />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SEND_PATH}
-          element={
-            <PrivateRoute>
-              <SendPayment />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SETTINGS_PATH}
-          element={
-            <PrivateRoute>
-              <Settings />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SIGN_MESSAGE_PATH}
-          element={
-            <PrivateRoute>
-              <SignMessage />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SHARE_NFT_PATH}
-          element={
-            <PrivateRoute>
-              <ShareNFT />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SHARE_PUBLIC_ADDRESS_PATH}
-          element={
-            <PrivateRoute>
-              <ShareAddress />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={SHARE_PUBLIC_KEY_PATH}
-          element={
-            <PrivateRoute>
-              <SharePublicKey />
-            </PrivateRoute>
-          }
-        />
-        <Route
-          path={ABOUT_PATH}
-          element={
-            <PrivateRoute>
-              <About />
-            </PrivateRoute>
-          }
-        />
-        <Route path={RESET_PASSWORD_PATH} element={<ResetPassword />} />
-        <Route
-          path={TRUSTED_APPS_PATH}
-          element={
-            <PrivateRoute>
-              <TrustedApps />
-            </PrivateRoute>
-          }
-        />
-      </SentryRoutes>
+      <Suspense fallback="loading">
```

Comment:
@FlorianBouron Here is the Suspense usage for making i18next work

Reply:
@FlorianBouron I can't put <Suspense> within <SentryRoutes>, the app would not render